### PR TITLE
Adding tests for Arithmetic

### PR DESCRIPTION
**Pull request by stefan-stefanooov**
**Description**:
Adding tests for Arithmetic

**Related issue(s)**:
https://github.com/hashgraph/hedera-smart-contracts/issues/473

### CODE REVIEW

File: test/solidity/cryptomath/arithmetic.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            const res = await contract.add()
+            await res.wait()
+        } catch (error) {
+            expect(error).to.exist
+            const name = await contract.checkName()
+            expect(name).to.equal('Arithmetic')
+        }
+    });
+
+    it('it should confirm solidity functionality: Arithmetic, checked overflow - confirm revert add2', async function () {
```
Comment (natanasow) [inline test/solidity/cryptomath/arithmetic.js:45]: same as above and all other tests below

File: contracts/solidity/cryptomath/Arithmetic.sol [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    uint256 minUint = type(uint256).min;
+
+    function checkName() external view returns (string memory){
+        return name;
+    }
+
+    function add() external returns (bool) {
+        name = "Arithmetic check if NOT reverted";
+        maxUint = maxUint + 1;
+        return true;
```
Comment (natanasow) [inline contracts/solidity/cryptomath/Arithmetic.sol:16]: the return statement could be removed if we don't use it

**Review by natanasow — CHANGES_REQUESTED** [review]
Some improvements IMO, @Nana-EC what do you think about that?

File: test/solidity/cryptomath/arithmetic.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    it('it should confirm solidity functionality: Arithmetic, checked overflow - confirm revert add', async function () {
+        try {
+            const res = await contract.add()
+            await res.wait()
+        } catch (error) {
+            expect(error).to.exist
+            const name = await contract.checkName()
+            expect(name).to.equal('Arithmetic')
+        }
+    });
```
[the remark this answers is not in the document]
Comment (Nana-EC) [reply test/solidity/cryptomath/arithmetic.js:43]: Agreed

**Review by Nana-EC — DISMISSED** [review]

**Comment by stefan-stefanooov** [comment]
@Nana-EC  please look at the `utils/helpers.js` changes.
I think they are needed. And they were breaking the tests constantly

**Review by Nana-EC — APPROVED** [review]

**Review by natanasow — APPROVED** [review]

**Pull request merged**